aixplain/factories/file_factory.py

```diff
@@ -104,3 +122,26 @@ def to_link(cls, data: Union[Text, Dict]) -> Union[Text, Dict]:
             if cls.check_storage_type(data) == StorageType.FILE:
                 data = cls.upload(local_path=data)
         return data
+
+    @classmethod
+    def create(
+        cls, local_path: Text, tags: Optional[List[Text]] = None, license: Optional[License] = None, is_temp: bool = True
+    ) -> Text:
+        """
+        Uploads a file to an S3 bucket.
+
+        Args:
+            local_path (Text): The local path of the file to upload.
+            tags (List[Text], optional): tags of the file
+            license (License, optional): the license for the file
+            is_temp (bool): specify if the file that will be upload is a temporary file
+
+        Returns:
+            Text: The S3 path where the file was uploaded.
+
+        Raises:
+            FileNotFoundError: If the local file is not found.
+            Exception: If the file size exceeds the maximum allowed size.
+        """
```

Comment:
Add an assertion that check whether license is not None if is_temp is not equals to False